content.md

```diff
@@ -327,6 +327,34 @@ Hi {github username}! You've successfully authenticated, but GitHub does not pro
 
 The [Ruby LSP](https://marketplace.visualstudio.com/items?itemName=Shopify.ruby-lsp) is an extension that provides performant rich features for Ruby. It connects to the ruby-lsp language server gem to analyze Ruby code and enhance the user experience
 
+## Customizing your terminal
+
```

Comment:
good start. It might be worth exploring how this works (and some potential risks) a bit. Eg setting up a .bash_profile or .zshrc file, how they're hidden by default (`ls -a`), how environment settings are applied at terminal start, and terminal environment debugging tips. Does that make sense?